region/maridia/inner-pink/Botwoon Energy Tank Room.json

```diff
@@ -881,7 +881,7 @@
       "link": [4, 1],
       "name": "Suitless Water Shinecharge",
```

Comment:
Tempting to call this Suitless Run Through.  The tech name being in the title is more important but its a tad confusing for those of us looking at the entrance condition